sneakers.gemspec

```diff
@@ -33,7 +33,7 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency 'rabbitmq_http_api_client'
   gem.add_development_dependency 'redis'
 
-  gem.add_development_dependency 'minitest', '~> 5.15'
+  gem.add_development_dependency 'minitest', '~> 5.15.0'
```

Comment:
It seems like newer versions of `minitest` was causing specs to fail in Ruby 2.6 and 2.7. For now, if we just restrict the `minitest` version specs will continue to pass, and Ruby 3.0 compatibility is not broken.